pkg/ingester/limiter_test.go

```diff
@@ -17,65 +18,124 @@ func TestSeriesLimit_maxSeriesPerMetric(t *testing.T) {
 		maxLocalSeriesPerMetric  int
 		maxGlobalSeriesPerMetric int
 		ringReplicationFactor    int
+		ringZoneAwarenessEnabled bool
```

Comment:
Would it be possible to refactor limiter code to avoid the same repeated tests? (In a follow-up PR, if at all).

Reply:
I think could be possible. Added to my TODO list for a separate PR.